contracts/LinniaRecords.sol

```diff
@@ -148,6 +154,57 @@ contract LinniaRecords is Ownable, Pausable, Destructible {
         return true;
     }
 
+    /* @dev allows a record owner to add a policy to a record as long as it conforms */
+    /* @param dataHash the record the owner is adding a policy to */
+    /* @param policy an address of a contract that conforms to the policy interface to add */
```

Comment:
consider rewording:
`an address of a contract that conforms to the policy interface to add`